setup.py

```diff
@@ -2,19 +2,42 @@
 
 from setuptools import setup
 
+import i18n
```

Comment:
Importing the product code into the setup.py is fragile.  I assume this is why you had to move the import into `__call__` in `i18n/__init__.py`.  It would be better to not do this.